stdlib/stream.mli

```diff
@@ -67,7 +67,8 @@ val iter : ('a -> unit) -> 'a t -> unit
 
 val next : 'a t -> 'a
 (** Return the first element of the stream and remove it from the
-   stream. Raise {!Stream.Failure} if the stream is empty. *)
+   stream.
+   @raise Stream.Failure if the stream is empty. *)
```

Comment:
Failure ? or Stream.Failure. Their is another one.

Reply:
I would put `Stream.Failure` so that the text which will be displayed now is clear. https://github.com/ocaml/odoc/issues/440 should deal with this in odoc ultimately.